CHANGELOG.md

```diff
@@ -15,6 +15,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 - Precursor charges are exported as integers instead of floats in the mzTab output file, in compliance with the mzTab specification.
 
+### Removed
+
+- Removed the `save_top_k` option from the Casanovo config, the model with the lowest validation loss during training will now be saved to a fixed filename `best.ckpt`. 
```

Comment:
**todo:** Sorry for not catching this earlier, but it should not actually be `best.ckpt`, but rather `<prefix>.best.ckpt` if an output name is specified, as per our discussion [here](https://docs.google.com/document/d/1szkb7l8Obm01UESl8UNj53Dm4W3Y7BG_lO1eHX4H_LA/edit). The final API will still be a bit different, with both a directory and a file root that can be specified, but I think we should already incorporate this to the extent possible in this PR.

Reply:
Sounds good, I added an option to the `ModelRunner` constructor, and I'll extend the functionality in future PRs.